MSU-Documentation.md

```diff
@@ -848,15 +848,15 @@ Checks if debugging is enabled for mod `_modID` and flag `_flagID`.
 
 - `::isDebugEnabled( _modID, _flagID = "default")`
 
-Convenience function for this.MSU.Debug.isEnabledForMod.
+Convenience function for this.MSU.Systems.Log.isEnabledForMod.
 
 - `::printLog( _printText, _modID, _flagID = null)`, `::printWarning`, `::printError`
 
 Substitutes for `this.logInfo`, `this.logWarning` and `this.logError`. Prints the log as `_printText` if debugging is enabled for the mod id `_modID`. `_flagID` specifies a flag of the mod, and is set to default if left empty.
 
 `Other debugging tools`
 
-- `this.MSU.Debug.printStackTrace( _maxDepth = 0, _maxLen = 10, _advanced = false )`
+- `this.MSU.Systems.Log.printStackTrace( _maxDepth = 0, _maxLen = 10, _advanced = false )`
```

Comment:
This isn't correct. The printStackTrace function is currently in `this.MSU.Debug` on your branch